reboot_toolkit/toolkit_backend.py

```diff
@@ -399,6 +403,168 @@ def load_games_to_df_from_s3_paths(
 
     return all_games_df
 
+def load_sampled_games_to_df_from_s3_paths(
+        game_paths: list[str], add_ik_joints: bool = False, add_elbow_var_val: bool = False, game_proportion: float = 1.0, movement_proportion: float = 0.2, min_movements: int = 0

```

Comment:
I know it's probably not the best coding practice, but to save us large data costs for now, rather than making game proportion and movement proportion optional input values, i would actually hard code internal to the function proportions that we think make sense. Like someone only wants to download a couple of games, we'll download the entirety of each game. But if someone wants to download 100 games, we'll automatically only download a sample

So in pseudo-code, I would so something like:

if len(game_paths) > some reasonably large number:
    game_proportion = some reasonable fraction
    movement_proportion = some reasonable fraction

Reply:
Hard coded movement_proportion and min movements. See below for comments on game_proportion